apps/gallery/js/ImageEditor.js

```diff
@@ -298,10 +300,16 @@ function editPhoto(n) {
     var imagesize = metadata.width * metadata.height;
     var maxsize = CONFIG_MAX_EDIT_PIXEL_SIZE || CONFIG_MAX_IMAGE_PIXEL_SIZE;
 
+    LazyLoader.load(['shared/js/media/jpeg-exif.js'], () => {
```

Comment:
I actually forfeit metadata on the cropResizeRotate() now. But I need to capture the Exif anyhow. So we parse it once, and here. And only if the blob is JPEG.